src/main/java/com/assylias/jbloomberg/IntradayTickData.java

```diff
@@ -4,10 +4,8 @@
  */
 package com.assylias.jbloomberg;
 
-import com.google.common.collect.LinkedListMultimap;
-import com.google.common.collect.Multimap;
-import com.google.common.collect.Table;
-import com.google.common.collect.TreeBasedTable;
+import com.google.common.collect.*;
+
```

Comment:
Please use single imports instead of wildcards to be consistent with the rest of the code.